Use logging instead of printing to stdout.

lucc opened the pull request:
This should address #22.  The PEP-8 commit is just an extra.

esalazar commented:
Hi @lucc, can you fix the conflicts. This looks good overall.

Also how did you go about fixing the pep-8 errors. Just want to make sure it wasn't done automatically as that might incur some issues.

lucc commented:
I updated it and removed the big pep8 commit. That one would have been a lot of work to rebase. It was originally done by manually fixing the errors reported by pycodechecker and friends.
